### PR TITLE
Hydro data updates

### PR DESCRIPTION
**Pull request by jhamman**
Adding two new datasets to the Pangeo catalog. Do not merge until I move these to their new bucket.

### CODE REVIEW

File: intake-catalogs/hydro.yaml [hunk 1]
```diff
@@ -60,3 +60,46 @@ sources:
...
+        type: str
+        default: TAXNWRB
+    metadata:
+      url: https://soilgrids.org/
+      tags: Soil
+    driver: rasterio
+    args:
+      urlpath: "gs://pangeo-data/soilgrids/{{ variable }}_250m.tif"
+      chunks: {'y': 5120, 'x': 5120}
+      storage_options: {'anon': True}  # TODO: remove, see GH#61
```
Comment (jhamman) [inline intake-catalogs/hydro.yaml]: @rabernat and @davidbrochart, wondering if you have suggestions on how to do this. This catalog loads fine in ocean.pangeo.io and fails on travis with the following error:

```
E   rasterio._err.CPLE_AppDefinedError: Fetching OAuth2 access code from auth code failed.
```

The bucket is currently public so I'm not sure why I need to do any auth here. Thoughts?

Comment (davidbrochart) [reply intake-catalogs/hydro.yaml]: We have a similar issue in #59.

Comment (davidbrochart) [reply intake-catalogs/hydro.yaml]: Actually this is a different issue. This one is more similar to https://github.com/intake/intake-xarray/issues/31. I made a PR here: https://github.com/jhamman/pangeo-datastore/pull/1

**Comment by jhamman** [comment]
Hi @charlesbluca - any chance you could look into this? I'm curious how we will go about testing things like this now that http://catalog.pangeo.io/ is up. I'm pretty sure the failures here are just related to the CI system we're using in this repo but it would be good to get your thoughts on them.

**Comment by charlesbluca** [comment]
It looks like the failures are rooted to a bad response from `http://metadata.google.internal/computeMetadata/v1/` - are the requests made to this URL being done by our own service account? The error page doesn't seem to mention anything about lack of permissions at this URL - just a 404 error. Is this form of refreshing service account info deprecated?

**Comment by rabernat** [comment]
The CI is broken here because we don’t have credentials in Travis to open requester pays bukets. 

Our options are:
- fix this (by providing credentials somehow)
- disable the test that actually opens the dataset with xarray and just focus on testing the intake catalog itself

I favor temporarily disabling the xarray opening, also because it’s slow. It doesn’t scale to open every dataset in the catalog every time CI is run. An even better option would be to look at the diff and only open the _new_ datasets in each PR. That would require some git ninja skills that I don’t have.

**Comment by jhamman** [comment]
FWIW: I am in favor of disabling the dataset open here and just testing / linting the catalog.

**Comment by charlesbluca** [comment]
If we want to leverage a solution that allows for additional credentials, we could consider a Cloud Function that tests updated catalogs (or just all catalogs on a trigger). This is currently how I've been updating the build badges for the catalog.

**Comment by rabernat** [comment]
@jhamman could you just rebase this on master. The tests should go green.

**Pull request merged**